unix/vncconfig/vncconfig.cxx

```diff
@@ -240,13 +240,19 @@ int main(int argc, char** argv)
   if (i < argc) {
     for (; i < argc; i++) {
       if (strcmp(argv[i], "-connect") == 0) {
+        Bool viewOnly = False;
         i++;
+        if (strcmp(argv[i], "-viewOnly") == 0 ||
+            strcmp(argv[i], "-view-only") == 0) {
```

Comment:
Sorry, I missed this the first rounds. But why two variants? And case-sensitive upper case seems a bit surprising.

Reply:
Yup, they are redundant. I must admit I forgot why I added both. It was probably because of some test script. I will change the code to use just `-view-only`.